projects/js-packages/components/changelog/update-i18n-number-formatters-numberFormat

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: deprecated
+
+Replace number-format component with number-formatters package
```

Comment:
I note there's also a removal in here, you can no longer `import { numberFormat } from '@automattic/jetpack-components';`.

That should probably be listed as a major-significance change in the changelog, e.g.
```suggestion
Significance: major
Type: removed

BREAKING: Removed `numberFormat` function. Use the `@automattic/number-formatters` package instead.
```

Reply:
Thanks @anomiex . Updated per suggestion.